2-Regression/1-Tools/translations/README.es.md

```diff
@@ -0,0 +1,206 @@
+# Comience con Python y Scikit-learn para modelos de regresión
+
+![Resumen de regresiones en un boceto](../../sketchnotes/ml-regression.png)
+
+> Boceto de [Tomomi Imura](https://www.twitter.com/girlie_mac)
+
+## [Cuestionario previo](https://jolly-sea-0a877260f.azurestaticapps.net/quiz/9/)
+## Introducción
+
+En estas cuatro lecciones, descubrirá como crear modelos de regresión. Discutiremos para que sirven estos en breve. Pero antes de hacer cualquier coasa, asegúrese de tener las herramientas adecuadas para comenzar el proceso!
+
+En esta lección, aprenderá a:
+
+- Configurar su computadora para tares locales de machine learning.
+- Trabajar con cuadernos Jupyter.
+- Usar Scikit-learn, incluida la instalación.
+- Explorar la regressión lineal con un ejercicio práctico.
+
+## Instalaciones y configuraciones.
+
+[![Uso de Python con Visual Studio Code](https://img.youtube.com/vi/7EXd4_ttIuw/0.jpg)](https://youtu.be/7EXd4_ttIuw "Uso de Python con Visual Studio Code")
```

Comment:
We recently updated this to use a video specific to this lesson (link is updated in the English version). Could you be so kind and update the link? I think the captions/text can stay the same.

```suggestion
[![Uso de Python con Visual Studio Code](https://img.youtube.com/vi/yyQM70vi7V8/0.jpg)](https://youtu.be/yyQM70vi7V8 "Uso de Python con Visual Studio Code")
```